src/EventListener/BlameListener.php

```diff
@@ -48,7 +48,10 @@ public function onKernelRequest(RequestEvent $event): void
         }
     }
 
-    public static function getSubscribedEvents()
+    /**
+     * @return array<string, string>
+     */
+    public static function getSubscribedEvents(): array
```

Comment:
you should not be added the native return type as this is a BC break if someone extends the class (which is neither final nor internal).

The comment from me that you were referring to only talks about adding the phpdoc.

Reply:
You're right @stof, I just removed native returns in order to avoid BCs.